### ISSUE TITLE
Keepass2Android keyboard not fully working

### ISSUE THREAD

**Issue by shompoe** [PhilippC/keepass2android#1716]
In Bromite the keepass2android keyboard only allows filling of username and password via the username and password buttons. Filling of other fields via the android button does not work.

**Comment by shompoe** [comment]
Update:  this appears to actually be an issue running on GrapheneOS.   I have tried Bromite browser, Vanadium browser and Brave browser and results are the same on all 3.

The keyboard with the KeepassDX app works perfectly and therein lies the answer to solving this problem.
